src/dmtcp_coordinator.cpp

```diff
@@ -281,9 +282,9 @@ DmtcpCoordinator::handleUserCommand(char cmd, DmtcpMessage *reply /*= NULL*/)
     JTRACE("blocking checkpoint beginning...");
     blockUntilDone = true;
     break;
-  case 'x': case 'X':  // prefix exit command, prior to checkpoint command
-    JTRACE("Will exit after creating the checkpoint...");
-    exitAfterCkptOnce = true;
+  case 'K': case 'x':  // prefix kill command, after ckpt cmd ('x' deprecated)
```

Comment:
Hi @gc00, I think there is a bug in this PR. If we type `kc` in the coordinator window, it would immediately kill the computation without checkpointing (because `k` is interpreted first). `xc` was a special case since `x` wasn't used anywhere else and so pairing it with `c` was ok.

However, with this new change, if we press `k` in the coordinator window, it wouldn't kill the computation immediately. 

Minor: you forgot to remove the other `x` :-).